htdocs/projet/list.php

```diff
@@ -147,8 +146,10 @@
 $search_date_end_end = dol_mktime(23, 59, 59, $search_date_end_endmonth, $search_date_end_endday, $search_date_end_endyear);	// Use tzserver
 $search_category_array = GETPOST("search_category_".Categorie::TYPE_PROJECT."_list", "array");
 
-if ($search_status == '') {
-	$search_status = -1; // -1 or 1
+if (is_array(GETPOST('search_status', 'none'))) {	// 'none' because we want to know type before sanitizing
```

Comment:
You can use GETPOSTISARRAY('search_status') for that.

Reply:
DONE